src/Magento/FunctionalTestingFramework/Test/Handlers/TestObjectHandler.php

```diff
@@ -102,28 +102,37 @@ public function getObject($testName)
      */
     public function getAllObjects()
     {
+        $errCount = 0;
         $testObjects = [];
         foreach ($this->tests as $testName => $test) {
             try {
                 $testObjects[$testName] = $this->extendTest($test);
             } catch (FastFailException $exception) {
                 throw $exception;
             } catch (\Exception $exception) {
+                $errCount++;
                 LoggingUtil::getInstance()->getLogger(self::class)->error(
-                    "Unable to create test " . $testName . "\n" . $exception->getMessage()
+                    "Unable to extend test " . $testName . "\n" . $exception->getMessage()
                 );
-                if (MftfApplicationConfig::getConfig()->getPhase() == MftfApplicationConfig::GENERATION_PHASE) {
-                    print("ERROR: Unable to create test " . $testName . "\n" . $exception->getMessage());
-                }
                 if (MftfApplicationConfig::getConfig()->getPhase() != MftfApplicationConfig::EXECUTION_PHASE) {
                     GenerationErrorHandler::getInstance()->addError(
                         'test',
                         $testName,
-                        self::class . ': Unable to create test ' . $exception->getMessage()
+                        self::class . ': Unable to extend test ' . $exception->getMessage()
                     );
                 }
             }
         }
+
+        if ($errCount > 0
+            && MftfApplicationConfig::getConfig()->getPhase() == MftfApplicationConfig::GENERATION_PHASE) {
+            print(
+                "ERROR in TestObjectHandler::getAllObjects(): "
+                . strval($errCount)
+                . " Test(s) cannot to be extended. See mftf.log for details."
```

Comment:
grammer fix:
`Test(s) cannot to be extended. See mftf.log for details. `

Reply:
Fixed